tests/containers/single_container_webui.pm

```diff
@@ -9,7 +9,7 @@ sub run {
 
   assert_script_run("openssl req -newkey rsa:4096 -x509 -sha256 -days 365 -nodes -subj '/CN=www.mydom.com/O=My Company Name LTD./C=DE' -out server.crt -keyout server.key");
 
-  assert_script_run("docker run --rm -d --network testing $volumes $certificates -p 80:80 --name openqa_webui openqa_webui");
+  assert_script_run("docker run -d --network testing $volumes $certificates -p 80:80 --name openqa_webui openqa_webui");
```

Comment:
How does removing `--rm` give us logs? This *should* mean that the container isn't removed after the run, shouldn't it?

Reply:
--rm                                       Remove container (and pod if created) after exit

Reply:
You are right, it maintains the container created until it is destroyed manually using docker rm that happens in the last line of the run, or in the post_fail_hook if something is wrong

Reply:
About the logs, I add a new line to print the logs in case of failure. I am not sure if show the logs if all are correct and pass the test

Reply:
The *post_fail_hook* only applies if the test is considered failed, and I think that's the case here. If a container command fails the test also fails.